compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -827,9 +827,14 @@ extern "C" LLVMRustResult LLVMRustOptimize(
       !NoPrepopulatePasses) {
     PipelineStartEPCallbacks.push_back(
         [](ModulePassManager &MPM, OptimizationLevel Level) {
-          MPM.addPass(LowerTypeTestsPass(/*ExportSummary=*/nullptr,
-                                         /*ImportSummary=*/nullptr,
-                                         /*DropTypeTests=*/false));
+          MPM.addPass(LowerTypeTestsPass(
+              /*ExportSummary=*/nullptr,
+              /*ImportSummary=*/nullptr,
+#if LLVM_VERSION_GE(20, 0)
+              /*DropTypeTests=*/llvm::lowertypetests::DropTestKind::None));
+#else
+              /*DropTypeTests=*/false));
+#endif
```

Comment:
We could also omit this arg to get its default value on both versions, right? I'm not sure why it was explicitly written out.

Reply:
Yes, I think we could - as far as I can tell it's had the same default all the way back to LLVM 11. Do you have a preference between spelling it out and just picking up the default?

Reply:
I would rather use the default to avoid the version conditions, since there's no functional difference.

Reply:
Done